core/tests/data/third_party/css/third_party.css

```diff
@@ -25795,4 +25795,7 @@ img-crop{width:100%;height:100%;display:block;position:relative;overflow:hidden}
 
 .select2-container--classic.select2-container--open .select2-dropdown {
```

Comment:
I don't think this is the correct place to add this. This file doesn't get used in the main repo (it is only used in the tests). Please add this in the corresponding component's HTML file.

Reply:
To add style we need to use View encapsulation None is that okay because it's a material component and using ngdeep throws lint error